### PR TITLE
Update References and API usage

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

Follow up PR of https://github.com/DynamoDS/Dynamo/pull/9361

[DYN-1457](https://jira.autodesk.com/browse/DYN-1457)
As a Dynamo Client, I want my pre-release build with ASM beta to work with DynamoSandbox

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] Snapshot of UI changes, if any.

### Reviewers

@mjkkirschner @aparajit-pratap

### CODE REVIEW

**Comment by mjkkirschner** [comment]
Have you tested this locally @QilongTang? - Do you want to FYI Angelo to look at this usage?

**Comment by QilongTang** [comment]
@mjkkirschner Angelo knows the new API already, I am testing this in D4R with R2020 now

File: src/Config/packages.aget [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-      "DynamoVisualProgramming.DynamoServices" : "2.1.0-beta7020",
-      "DynamoVisualProgramming.Tests" : "2.1.0-beta7020",
-      "DynamoVisualProgramming.WpfUILibrary" : "2.1.0-beta7020",
-      "DynamoVisualProgramming.ZeroTouchLibrary" : "2.1.0-beta7020",
+      "DynamoVisualProgramming.Core" : "2.1.0-beta7321",
+      "DynamoVisualProgramming.DynamoCoreNodes" : "2.1.0-beta7321",
+      "DynamoVisualProgramming.DynamoServices" : "2.1.0-beta7321",
+      "DynamoVisualProgramming.Tests" : "2.1.0-beta7321",
+      "DynamoVisualProgramming.WpfUILibrary" : "2.1.0-beta7321",
+      "DynamoVisualProgramming.ZeroTouchLibrary" : "2.1.0-beta7321",
```
Comment (aparajit-pratap) [inline src/Config/packages.aget]: Are these based on the latest NuGet versions posted? Does this file need to be updated manually each time you post new NuGet's?

Comment (QilongTang) [reply src/Config/packages.aget]: @aparajit-pratap Yes, unfortunately Revit team changed from always consuming latest beta to fixed version to slow down the process and making sure the build is stable

**Comment by QilongTang** [comment]
@mjkkirschner @aparajit-pratap My local testing passed. Let me know if you have any comments.

File: src/DynamoRevit/DynamoRevit.cs [hunk 2]
```diff
@@ -481,8 +481,7 @@ internal static Version PreloadAsmFromRevit()
             Version libGversion = findRevitASMVersion(asmLocation);
 
             var dynCorePath = DynamoRevitApp.DynamoCorePath;
-            var libGFolderName = string.Format("libg_{0}_{1}_{2}", libGversion.Major, libGversion.Minor, libGversion.Build);
-            var preloaderLocation = Path.Combine(dynCorePath, libGFolderName);
+            var preloaderLocation = DynamoShapeManager.Utilities.GetLibGPreloaderLocation(libGversion, dynCorePath);
```
Comment (mjkkirschner) [inline src/DynamoRevit/DynamoRevit.cs]: @QilongTang would you mind adding a comment here to briefly document the fallback behavior? - Or does this method have a summary tag that shows up?

**Comment by mjkkirschner** [comment]
LGTM

File: src/DynamoRevit/DynamoRevit.cs (hunk 2 above)
Comment (QilongTang) [reply src/DynamoRevit/DynamoRevit.cs]: The summary does not show up for me in my VS, I can add comments but will be the same with function signature

**Comment by QilongTang** [comment]
Added Comments. Merging

**Comment by QilongTang** [comment]
FYI: @ZiyunShang  @AndyDu1985

**Pull request merged**